### ISSUE TITLE
SWADE Duplicate Dialog Box on Right Click HUD

### ISSUE THREAD

**Issue by Neurochimp** [Drental/fvtt-tokenactionhud#354]
Hello,

I was tracking down an issue with the interactions between SWADE Tools & Token Action HUD yesterday. We solved the initial issue with SWADE Tools from the HUD - simple [SWADE Tools] & complex [standard] roll/dialog boxes populate on left click of HUD.

Finding after that was solved there is still a lingering problem. When you right click on the HUD, it will fire off the request twice for standard roll/dialog box. The dev there mentioned it after solving the SWADE Tools side & can confirm if I only have the HUD enabled, this behavior occurs.

When you get a chance, can you take a look at it? It's not game breaking just tad annoying.

Thank you.
![Duplicate_Dialog Box](https://user-images.githubusercontent.com/116909768/206911756-a9e4125f-7e30-46e6-98cb-a3be0f7b1d85.png)

**Comment by Larkinabout** [comment]
Hi @Neurochimp, I have been unable to reproduce this issue on my end with Token Action HUD 3.0.17 and SWADE Tools 1.10.15:
![swade-swade-tools](https://user-images.githubusercontent.com/105953297/211179291-e188df8f-487d-4918-8e4f-ca5dbee919a7.gif)
Is this issue now resolved for you?

**Comment by Neurochimp** [comment]
I was able to verify it when I submitted the ticket. Not sure if something else has changed that may have solved it - such as CORE update.
I'm currently using 3.015 due to the other bug we noticed that player characters aren't showing the HUD if they have powers - ticket #359. This ticket was really just a minor annoyance, as my group doesn't use the right click function. If I see it again & can replicate it, I'll bring it back here.
